src/scenes/Projects/List/ProjectList.tsx

```diff
@@ -20,25 +21,46 @@ const useStyles = makeStyles(({ spacing }) => ({
   options: {
     margin: spacing(3, 0),
   },
+  listContainer: {
+    height: '100%',
+  },
   projectItem: {
     marginBottom: spacing(2),
   },
 }));
 
 export const ProjectList: FC = () => {
+  const classes = useStyles();
   const formatNumber = useNumberFormatter();
   const sort = useSort<Project>();
   const [filters, setFilters] = useProjectFilters();
+  const [itemsPerPage, setContainerHeight] = useItemsPerPage(240);
 
-  const { data } = useProjectListQuery({
+  const { data, fetchMore } = useProjectListQuery({
     variables: {
       input: {
         ...sort.value,
         filter: filters,
+        count: itemsPerPage,
       },
     },
   });
-  const classes = useStyles();
+
+  const hasMore = data?.projects.hasMore ?? false;
+  const currentItemsCount = data?.projects.items.length ?? itemsPerPage;
+
+  const loadMoreItems = async () => {
+    await fetchMore({
+      variables: {
+        input: {
+          ...sort.value,
+          filter: filters,
+          count: itemsPerPage,
+          page: Math.floor(currentItemsCount / itemsPerPage + 1),
+        },
+      },
+    });
+  };
```

Comment:
All of this could be put into a `useInfiniteList` hook that returns props specifically matching what `VirtualList` is expecting.